src/main/kotlin/com/romainpiel/catsanddogs/Main.kt

```diff
@@ -25,34 +26,34 @@ fun main(args: Array<String>) {
 
             // deprecated
             get("/schedule.json") {
-                val parameters = parameters(call.request.queryParameters)
+                val from = offsetDateTime(call.request.queryParameters)
                 val conference = Conference.MCE4
+                val acceptLanguage = "pl-PL"
+                val locale = Locale.forLanguageTag(acceptLanguage)
 
-                val json = scheduleRepository.schedule(parameters.from, parameters.locale, conference).blockingGet()
+                val json = scheduleRepository.schedule(from, locale, conference).blockingGet()
                 call.respondText(json, ContentType.Application.Json)
             }
 
             get("/{conference}/schedule.json") {
-                val parameters = parameters(call.request.queryParameters)
+                val from = offsetDateTime(call.request.queryParameters)
                 val conference: Conference = Conference.instance(call.parameters["conference"]) ?: Conference.MCE4
+                val acceptLanguage: String = call.request.headers["Accept-Language"] ?: "pl-PL"
+                val locale = Locale.forLanguageTag(acceptLanguage)
```

Comment:
weird that it doesn't work, I'll have a look this weekend

Reply:
See https://github.com/RomainPiel/CatsAndDogs-kotlin-bff/pull/3#discussion_r136696241